addons/udes_stock/models/stock_picking.py

```diff
@@ -89,34 +92,36 @@ def _requires_backorder(self, mls):
                 return True
         return False
 
-    def _prepare_picking_info(self, picking_type, products_info=None, **kwargs):
+    def _prepare_picking_info(self, picking_type, products_info=False, **kwargs):
         """ Prepare the picking_info and products_info 
         :args:
             - picking_type: picking type of the picking
         
         :kwargs:
-            - products_info: list of dicts with product information
+            - products_info: False or list of dicts with product information
         
         :returns:
-            picking_values:
-            products_info
+            picking_values: list(dict) of picking values
+            products_info: False if products_info is none, or list(list(dict)) of product, qty info
         """
         picking_values = {
             'picking_type_id': picking_type.id,
             'location_id': picking_type.default_location_src_id.id,
             'location_dest_id': picking_type.default_location_dest_id.id,
         }
         picking_values.update(kwargs)
-
-        if any(isinstance(el, list) for el in products_info):
-            num_pickings = len(products_info)
-            picking_vals = [picking_values.copy() for i in range(num_pickings)]
+        if products_info is None:
```

Comment:
Are we sure we want to check if  `products_info` is None? default value is now `False`

Reply:
fixed